scripts/stability.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+const fs = require('fs');
+const packageJsonPath = process.argv[2] || path.join(process.cwd(), 'package.json');
+const package = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'));
```

Comment:
You could `require` the JSON file instead of doing the reading + parsing dance. It looks cleaner, too.

Reply:
require is relative to script path. I want relative to execution path.

Reply:
You can pass it an absolute path and it'll use that... So if you just `path.resolve`...